### PR TITLE
feat: always record direct_url.json

### PR DESCRIPTION
**Pull request by nichmor**
closes: #179

### CODE REVIEW

File: crates/rip_bin/src/main.rs [hunk 1]
```diff
@@ -301,11 +301,18 @@ async fn actual_main() -> miette::Result<()> {
                 console::style(pinned_package.version).italic()
             );
             let artifact_info = pinned_package.artifacts.first().unwrap();
-            let artifact = package_db
+            let (artifact, direct_url_json) = package_db
                 .get_wheel(artifact_info, Some(&wheel_builder))
                 .await?;
-            venv.install_wheel(&artifact, &UnpackWheelOptions::default())
-                .into_diagnostic()?;
+            println!("DIRECT URL JSON IS {:?}", direct_url_json);
```
Comment (wolfv) [inline crates/rip_bin/src/main.rs:307]: 👀

File: crates/rattler_installs_packages/src/index/package_database.rs [hunk 2]
```diff
@@ -263,15 +269,45 @@ impl PackageDb {
                     .get_cached_artifact::<SDist>(artifact_info, CacheMode::Default)
                     .await?;
 
-                return builder.build_wheel(&sdist).await.into_diagnostic();
+                let whl = builder
+                    .build_wheel(&sdist)
+                    .await
+                    .into_diagnostic()
+                    .expect("cannot build wheel");
```
Comment (wolfv) [inline crates/rattler_installs_packages/src/index/package_database.rs:276]: Should probably not be an `expect`, right?

**Review by wolfv — APPROVED** [review]

File: crates/rattler_installs_packages/src/index/package_database.rs (hunk 2 above)
Comment (nichmor) [reply crates/rattler_installs_packages/src/index/package_database.rs]: yes, should be ?
thanks for spotting it!

File: crates/rip_bin/src/main.rs (hunk 1 above)
Comment (nichmor) [reply crates/rip_bin/src/main.rs]: 🧹

**Review by tdejager — APPROVED** [review]

**Pull request merged**